src/components/KAlert/KAlert.vue

```diff
@@ -368,227 +373,234 @@ export const appearances: AlertAppearanceRecord = {
+  padding: var(--kui-space-0, $kui-space-0);
 }
 </style>
 
 <style lang="scss">
 @import '@/styles/variables';
+@import '@/styles/tmp-variables';
 @import '@/styles/functions';
 
 .k-alert {
   button.close > .kong-icon {
     &.info svg {
-      stroke: var(--KAlertInfoColor, var(--blue-500, color(blue-500)));
+      stroke: var(--KAlertInfoColor, var(--blue-500, var(--kui-color-text-primary, $kui-color-text-primary)));
     }
 
     &.success svg {
-      stroke: var(--KAlertSuccessColor, var(--green-600, color(green-600)));
+      stroke: var(--KAlertSuccessColor, var(--green-600, $tmp-color-green-600));
     }
 
     &.danger svg {
-      stroke: var(--KAlertDangerColor, var(--red-700, color(red-700)));
+      stroke: var(--KAlertDangerColor, var(--red-700, var(--kui-color-text-danger, $kui-color-text-danger)));
     }
 
     &.warning svg {
-      stroke: var(--KAlertWarningColor, var(--yellow-500, color(yellow-500)));
+      stroke: var(--KAlertWarningColor, var(--yellow-500, $tmp-color-yellow-500));
     }
   }
 
   .k-alert-action {
     & button {
-      font-size: 13px;
-      font-weight: 400;
+      font-size: var(--kui-font-size-20, $kui-font-size-20);
+      font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
       height: 30px;
-      line-height: 13px;
+      line-height: var(--kui-line-height-10, $kui-line-height-10);
 
       &:not(:first-of-type) {
-        margin-left: var(--spacing-sm);
+        margin-left: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
       }
     }
 
     &.info button.primary {
-      --KButtonPrimaryBase: var(--blue-500);
-      --KButtonPrimaryHover: var(--blue-200);
-      background-color: var(--blue-100);
-      color: var(--blue-500);
+      --KButtonPrimaryBase: var(--blue-500, var(--kui-color-background-primary, $kui-color-background-primary));
+      --KButtonPrimaryHover: var(--blue-200, var(--kui-color-background-primary-weaker, $kui-color-background-primary-weaker));
+      background-color: var(--blue-100, var(--kui-color-background-primary-weakest, $kui-color-background-primary-weakest));
+      color: var(--blue-500, var(--kui-color-text-primary, $kui-color-text-primary));
     }
 
     &.info button.outline {
-      --KButtonOutlineBorder: var(--blue-500);
-      --KButtonOutlineHoverBorder: var(--blue-600);
-      --KButtonOutlineActive: var(--blue-100);
-      --KButtonOutlineActiveBorder: var(--blue-500);
-      border: 1px solid var(--blue-400);
-      color: var(--blue-500);
+      --KButtonOutlineBorder: var(--blue-500, var(--kui-color-border-primary, $kui-color-border-primary));
+      --KButtonOutlineHoverBorder: var(--blue-600, var(--kui-color-border-primary-strong, $kui-color-border-primary-strong));
+      --KButtonOutlineActive: var(--blue-100, var(--kui-color-background-primary-weakest, $kui-color-background-primary-weakest));
+      --KButtonOutlineActiveBorder: var(--blue-500, var(--kui-color-border-primary, $kui-color-border-primary));
```

Comment:
comment(non-blocking): I guess we'll handle these theming variable overrides when we just remove them in the breaking release

Reply:
Yup that's what I thought too